mlir/test/Conversion/AIRToAIE/air_channel_to_objectFifo_L1toL1.mlir

```diff
@@ -0,0 +1,79 @@
+//===- air_channel_to_objectFifo_L1toL1.mlir --------------------*- MLIR -*-===//
+//
+// Copyright (C) 2022, Advanced Micro Devices, Inc.
+// SPDX-License-Identifier: MIT
+//
+//===----------------------------------------------------------------------===//
+
+// RUN: air-opt %s -air-to-aie='test-patterns=specialize-affine-if,lower-air-channels' | air-opt --canonicalize | FileCheck %s
```

Comment:
Test could be simplified by using the output of `specialize-affine-if` instead of running it here.

Reply:
I changed the test so it is the output of `specialize-affine-if` and also `canonicalize`.